agent/containermetadata/metadata_manager.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
That's not a convention, just a sign of us being immature golang coders when that code was written. I don't see a strong argument against renaming this. Please rename both as per my earlier comment.